src/utilities/__tests__/platformServices.test.ts

```diff
@@ -0,0 +1,53 @@
+import { authenticateUser } from '../platformServices';
+
+describe('Authenticate User method', () => {
+  it('should return a promise with user data', () => {
+    window.insights = {
+      chrome: {
+        auth: {
+          getUser: jest.fn().mockImplementation(
+            () =>
```

Comment:
Seems like all the code around this function could be happening just once for the whole file in a before block, with just this function being defined independently for each test.

Reply:
OK I pushed an update for that.

Reply:
Could we get this to something more like the following:
```ts
let getUser;

beforeEach(() => {
  jest.clearAllMocks();
  window.insights = {
    chrome: {
      auth: {
        // eslint-disable-next-line @typescript-eslint/no-empty-function
        getUser: jest.fn().mockImplementation(
          new Promise((resolve, reject) => {
            process.nextTick(getUser(resolve, reject));
          })
        )
      }
    }
  };
});

describe('Authenticate User method', () => {
  getUser = (resolve, reject) => { reject({ error: 'Error getting user' }) };

  it('should return a promise with user data', () => {
```

Just trying to isolate the part that actually changes.

Reply:
ah ok, so I had based these off the tests in swatch for similar methods, but there is actually a Jest helper for this situation that helps clean it up a lot - jest.fn().mockResolvedValue and jest.fn().mockRejectedValue.  I had tried them before but was running into issues but got them to work this time.  Just pushed a fix and it's much cleaner now.

Reply:
Yes, I like these other mocks more.  Thanks!